auth/src/charm.py

```diff
@@ -125,9 +125,8 @@ def _on_collect_unit_status(self, e: CollectStatusEvent):
                 "database config": self.database_config,
                 "backend gRPC endpoint": self.backend_grpc_endpoint,
             },
+            block_if_pebble_checks_failing={LitmusAuth.name: [LitmusAuth.name]},
```

Comment:
Makes me realize, we're using `LitmusAuth.name` as name of a bunch of things. The container, the pebble layer, and now a pebble check. Perhaps it would be wise to have LitmusAuth.container_name, LitmusAuth.layer_name, etc... in case we want to differentiate them in the future?
Also, "auth" is not a great pebble check name. "auth-server-up" could be?
Also suppose we want to add another pebble check. What do we call it? How do we extend this logic while keeping it obvious what's going on?

This is what I'd like to read:
```suggestion
            block_if_pebble_checks_failing={LitmusAuth.container_name: LitmusAuth.all_pebble_checks},
```